src/std/text/_zlib.scm

```diff
@@ -96,7 +96,7 @@ static ___SCMOBJ ffi_free (void *ptr)
 
 static z_stream *ffi_make_z_stream ()
 {
- z_stream *zs = malloc (sizeof (z_stream));
+ z_stream *zs = (z_stream *)malloc (sizeof (z_stream));
```

Comment:
no, it really sucks! I keep getting burned from this.

Reply:
aaand yup :)